src/chi-vue/src/components/data-table/DataTable.tsx

```diff
@@ -1286,21 +1287,142 @@ export default class DataTable extends Vue {
     }
   }
 
+  _printHead() {
+    return (
+      <thead>
+        <tr>
+          {Object.keys(this.data.head).map((column: string) => {
+            return <th>{this.data.head[column].label}</th>;
+          })}
+        </tr>
+      </thead>
+    );
+  }
+
+  _printFooter() {
+    const resultsCount =
+      this.mode === DataTableModes.CLIENT
+        ? this._showSelectedOnly
+          ? this._getSelectedFirstLevelRowsCount()
+          : this.data.body.length
+        : this.config.pagination.results;
+
+    return <div class={`${DATA_TABLE_CLASSES.PRINT_FOOTER}`}>{resultsCount} results</div>;
+  }
+
+  _printBody() {
+    if (this.data.body.length > 0) {
+      const bodyRows = this._sortedData && this._sortedData.length > 0 ? this._sortedData : this._serializedDataBody;
+
+      return (
+        <tbody>
+          {bodyRows?.map((bodyRow: DataTableRow) => {
+            return this._printRow(bodyRow, 'parent');
+          })}
+        </tbody>
+      );
+    }
+    return (
+      <tbody>
+        <tr>
+          <td colspan={Object.keys(this.data.head).length} class={DATA_TABLE_CLASSES.EMPTY}>
+            {this.config.noResultsMessage ? this.config.noResultsMessage : 'No results found'}
```

Comment:
`No results found` message is repeated in the file. Please, define a constant for it.